Add blog post for sprint 148 request page redesign (#427)
openSUSE/obs-landing: _posts/development/2023-09-07-request-workflow-redesign.md

---
layout: post
title: Request Page Redesign - Build Results Summary Gets Into Shape
category: development
---

> Last days, we have been working again on the request workflow redesign.

Reviewer: ```suggestion
Lately, we have been working again on the request workflow redesign.
```

> We introduce the build results summary of a request and add a light improvement to the information returned of its mentioned issues.

Reviewer: ```suggestion
We have introduced the build results summary of a request and improved the information displayed in the Mentioned Issues section.
```

All these improvements will help you focus on what matters and better digest all the information you receive when you review or visit a request.

The request redesign is part of [the beta program](/2018/10/04/the-beta-program/).

{% include partials/_series-of-posts-about-request-redesign.md %}

### Build Results Summary of a Request

Showing what is relevant at a glance conflicts with the nature and size of the data you are able to assimilate.
There is no "one size that fits all" solution.
That is why we have decided for a more flexible approach.

> Apart from the detailed view of build results of a request in its own tab, you now also become a summary of the Build Results in the very first Conversation tab.

Reviewer: ```suggestion
You still can see the detailed view in the Build Results tab but, from now on, you can also see a summary of the build results in the very first Conversation tab.
```
